src/com/facebook/buck/swift/SwiftDescriptions.java

```diff
@@ -70,4 +71,15 @@ private SwiftDescriptions() {
     output.bridgingHeader = args.bridgingHeader;
   }
 
+  static boolean isSharedRequested(Linker.LinkableDepType type) {
```

Comment:
hmm, i think i wont use the comparison, Apple could be crazy enough to have another type of dynamic library in the future.